core/includes/update.inc

```diff
@@ -418,6 +418,23 @@ function update_variable_del($name) {
 }
 
 /**
+ * Merge any new default values into the active configuration.
+ *
+ * Used by the update.php script to apply new default config data
+ * from updated modules.
+ *
+ * @param array $modules
+ *   An array where the keys are the modules with pending updates and the
+ *   values are an array of configuration files that require updates.
+ */
+function update_config_defaults($modules, &$context) {
+  foreach($modules as $module_name => $module_data) {
+    config_update_default_config($module_name);
+    $context['message'] = 'Updated default settings for ' . $module_name . '.';
```

Comment:
I was following the pattern that `update_do_one()` uses, which didn't t() the message. (See line ~539.)  I'll add a patch to fix both t() spots and move this one outside the loop.